MapboxPlugins.podspec

```diff
@@ -30,13 +30,13 @@ s.social_media_url = "https://twitter.com/mapbox"
 
   # ――― Source Location ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
-  s.source       = { :git => "https://github.com/mapbox/mapbox-plugins-ios.git", :branch => "traffic-plugin" }
+  s.source       = { :git => "https://github.com/mapbox/mapbox-plugins-ios.git", :branch => "master" }
 
   s.module_name = 'MapboxPlugins'
 
   s.subspec 'PluginKit' do |core|
     core.source_files  = 'PluginKit/*.h'
-    core.dependency 'Mapbox-iOS-SDK', '~> 3.7.0'
+    core.dependency 'Mapbox-iOS-SDK', '~> 4.1.0'
```

Comment:
Let’s have this plugin depend on `~> 4.0`. That way, folks who need to use v4.0._x_ for some reason can still use this plugin, while everyone else will be able to use it with v4._x_.